packages/notifi-react/lib/context/NotifiTargetContext.tsx

```diff
@@ -548,9 +653,49 @@ export const NotifiTargetContextProvider: FC<PropsWithChildren> = ({
     [],
   );
 
+  const refreshWeb3Target = useCallback(
+    async (web3Target?: Types.Web3TargetFragmentFragment) => {
+      if (web3Target) {
+        switch (web3Target.isConfirmed) {
+          case false:
+            updateTargetInfoPrompt('wallet', {
+              type: 'cta',
+              message: 'Sign Wallet',
+              onClick: () => {
+                // TODO: add action
```

Comment:
Shall we add `action` here?

Reply:
@ankush-notifi  can you address this one?